target/linux/mediatek/image/mt7622.mk

```diff
@@ -252,11 +252,11 @@ define Device/ubnt_unifi-6-lr-v1
 endef
 TARGET_DEVICES += ubnt_unifi-6-lr-v1
 
-define Device/ubnt_unifi-6-lr-ubootmod
+define Device/ubnt_unifi-6-lr-v1-ubootmod
   DEVICE_VENDOR := Ubiquiti
-  DEVICE_MODEL := UniFi 6 LR
+  DEVICE_MODEL := UniFi 6 LR v1
```

Comment:
`v1` should be part of `DEVICE_VARIANT`.

Reply:
thanks, fixing